site/content/docs/11-deploying.md

```diff
@@ -4,28 +4,25 @@ title: Deployment
 
 Sapper apps run anywhere that supports Node 8 or higher.
 
+### Deploying to Vercel ([formerly ZEIT Now](https://vercel.com/blog/zeit-is-now-vercel))
 
-### Deploying to Now
+#### Using Now v2
```

Comment:
Shouldn't this say Vercel?

Reply:
@benmccann product name is now, company name is Vercel :)

Reply:
Okay, looks like I'm wrong. They seem to have renamed the product after the company. I'm so confused.

Reply:
I also am very confused about this.
The product is called "Vercel". [The CLI also uses this name after the update](https://vercel.com/docs/v2/build-step).
The [domains continue to use `.now.sh`](https://vercel.com/docs/v2/custom-domains) 

However I would say

> Link your GitHub account with your personal account or team on Vercel from your dashboard. [...] Then, click "Install Now for GitHub".

Makes `now` the winner for... now.
Opinions?

Reply:
Okay updated opinion:
It's likely that

> On the installation page for our command-line interface, the vercel package is now recommended instead of the legacy now package
[Source: Changelog May](https://vercel.com/blog/changelog-may-2020#dashboard)

has the answer.
I would say we should change the heading to `vercel` as the package for `now` seems to be legacy.

Then again, the third party library probably needs to be updated or forked to support vercel, but this is not a sapper task, so it should be seen asa different task unrelated to this change imo.

Reply:
Yeah, I've just asked them on twitter.

I'm a maintainer of the third-party library so renaming that is no problem if @thgh agrees.

Reply:
It looks like the Vercel documentation is outdated. I filed an issue with them: https://github.com/zeit/docs/issues/1842